packages/teams-js/test/public/dialog.spec.ts

```diff
@@ -95,17 +95,17 @@ describe('Dialog', () => {
 
         it(`FRAMED: should initiate the post message to dialog using returned function. context: ${context}`, async () => {
           await framedMock.initializeWithContext(context);
-          const sendMessageToDialogHandler = dialog.open(dialogInfo, emptyCallback, emptyCallback);
-          sendMessageToDialogHandler('exampleMessage');
+          dialog.open(dialogInfo, emptyCallback, emptyCallback);
+          dialog.sendMessageToDialog('exampleMessage');
           const message = framedMock.findMessageByFunc('messageForChild');
           expect(message).not.toBeUndefined();
           expect(message.args).toStrictEqual(['exampleMessage']);
         });
 
         it(`FRAMELESS: should initiate the post message to dialog using returned function. context: ${context}`, async () => {
```

Comment:
We can change the test description according to new functionality :)

Reply:
And, I think, for unit testing, we don't need to open a dialog to test `sendMessageToDialog`